ha/cli/commands.py

```diff
@@ -105,18 +105,40 @@ def add_args(component_parser):
         start_parser.add_argument("service_name",
             help="Service name", action="store")
         start_parser.add_argument("--node",
-            help="Node name")
+            help="Provide node name to start service on particular node")
 
         stop_parser = action_subparser.add_parser("stop",
             help = "Stop Service")
         stop_parser.add_argument("service_name",
             help="Service name", action="store")
         stop_parser.add_argument("--node",
-            help="Node name")
+            help="Provide node name to stop service on particular node")
 
         status_parser = action_subparser.add_parser("status",
             help = "Status of service")
         status_parser.add_argument("service_name",
             help="Service name", action="store")
         status_parser.add_argument("--node",
-            help="Node name")
\ No newline at end of file
+            help="Provide node name to get status of service.")
+
+class SupportBundleCommand(Command):
+    @staticmethod
+    def add_args(component_parser):
+        """
+        Add parser for support bundle command
+
+        Args:
+            component_parser (<class 'argparse._SubParsersAction'>): argparse to add subparser.
+        """
+        bundle_parser =  component_parser.add_parser("support_bundle",
+                                help = "Manage support bundle operation like create, delete, show.")
```

Comment:
Do we support support bundle delete and show too?

Reply:
Not supported yet 
in option only create will show

Reply:
Removed other operation